redwood-compose/src/commonMain/kotlin/app/cash/redwood/compose/RedwoodComposition.kt

```diff
@@ -185,9 +185,9 @@ public interface RedwoodApplier<W : Any> {
  */
 @Composable
 @RedwoodCodegenApi
-public inline fun <P : Widget.Provider<*>, W : Widget<*>> RedwoodComposeNode(
+public inline fun <P : Widget.Provider<V>, W : Widget<V>, V : Any> RedwoodComposeNode(
```

Comment:
These generics are brutal, but it's for generated code only so whatever. This change is entirely so that the singleton `SetModifiers` function (in next file) works correctly in the generated code. As a bonus, we lose a cast here!